Index: BlockStruct/main.go
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package main\r\n\r\nimport \"./BLC\"\r\n\r\nfunc main() {\r\n\t//block := BLC.NewBlock(1,nil,[]byte(\"First Block\"))\r\n\t//bc  := BLC.CteateBlockChain()\r\n\t//bc = BLC.ReturnBlockOBJ()\r\n\t//bc.AddBlock([]byte(\"secend\"))\r\n\t//bc.AddBlock([]byte(\"three\"))\r\n\t//BLC.ReturnTheChain(bc)\r\n\t//bc.PrintChain()\r\n\r\n\t//for _, block := range bc.Blocks {\r\n\t//fmt.Printf(\"Block :%d Hash: %X\\n\", block.Height, block.Hash)\r\n\t//}\r\n\r\n\tcli := BLC.CLI{}\r\n\tcli.Run()\r\n\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/BlockStruct/main.go b/BlockStruct/main.go
--- a/BlockStruct/main.go	
+++ b/BlockStruct/main.go	
@@ -18,4 +18,5 @@
 	cli := BLC.CLI{}
 	cli.Run()
 
+
 }
Index: .idea/blockchain.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/blockchain.iml b/.idea/blockchain.iml
new file mode 100644
--- /dev/null	
+++ b/.idea/blockchain.iml	
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="WEB_MODULE" version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$">
+      <excludeFolder url="file://$MODULE_DIR$/BlockStruct" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
\ No newline at end of file
Index: .idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/modules.xml b/.idea/modules.xml
new file mode 100644
--- /dev/null	
+++ b/.idea/modules.xml	
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/blockchain.iml" filepath="$PROJECT_DIR$/.idea/blockchain.iml" />
+    </modules>
+  </component>
+</project>
\ No newline at end of file
Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/vcs.xml b/.idea/vcs.xml
new file mode 100644
--- /dev/null	
+++ b/.idea/vcs.xml	
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+    <mapping directory="$PROJECT_DIR$/src/github.com/boltdb/bolt" vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
